Rakefile

```diff
@@ -2,6 +2,10 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require_relative 'config/application'
+require 'rubocop'
+require 'rubocop/rake_task'
+# require 'rubocop-rake'
+require 'rubocop-rails'
```

Comment:
Looks like this is not part of the devise configuration.

I wonder why do we need them 😄 
And if we really want to keep this change. WDYT about moving this to another PR for describing the motivation of this change?